src/config/index.js

```diff
@@ -1,6 +1,8 @@
-const registerConfig = require('windbreaker-service-util/config')
+const configUtil = require('windbreaker-service-util/config')
 
 const Config = require('./Config')
 const config = module.exports = new Config()
 
-registerConfig(config)
+module.exports.load = async () => {
+  await configUtil.load({ config })
```

Comment:
There's more work that could be done here to add support for env based config loading. @Epitrochoid you should check out the work @Druotic did for that in core.